### PR TITLE
[Filebeat] Add Cisco ASA message '302023' parsing

### PR DESCRIPTION
**Pull request by chifu1234**
cisco/add adding message id 302023

## What does this PR do?

This PR will add parsing for cisco asa message id 302023.

## Why is it important?

This will add a common message for cisco asa clusters

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have made corresponding change to the default configuration files
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Author's Checklist

- [ ]

## Related issues

- 

## Logs

 
--
%ASA-6-302022: Built backup stub TCP connection for INTERNET:1.1.1.1/57475 (1.1.1.1/57475) to INTERN:1.2.3.4/443 (1.2.3.4/443)
  |

### CODE REVIEW

File: x-pack/filebeat/module/cisco/shared/ingest/asa-ftd-pipeline.yml [hunk 1]
```diff
@@ -318,6 +318,10 @@ processors:
       if: "ctx._temp_.cisco.message_id == '302022'"
       field: "message"
       pattern: "Built %{} stub %{network.transport} connection for %{_temp_.cisco.source_interface}:%{source.address}/%{source.port} to %{_temp_.cisco.destination_interface}:%{destination.address}/%{destination.port}"
+  - dissect:
+      if: "ctx._temp_.cisco.message_id == '302023'"
+      field: "message"
+      pattern: "Teardown stub %{network.transport} connection for %{_temp_.cisco.source_interface}:%{source.address}/%{source.port} to %{_temp_.cisco.destination_interface}:%{destination.address}/%{destination.port} duration %{_temp_.duration_hms} forwarded bytes %{:network.bytes}"
```
Comment (andrewkroh) [inline x-pack/filebeat/module/cisco/shared/ingest/asa-ftd-pipeline.yml:324]: There's an extra colon here.

```suggestion
      pattern: "Teardown stub %{network.transport} connection for %{_temp_.cisco.source_interface}:%{source.address}/%{source.port} to %{_temp_.cisco.destination_interface}:%{destination.address}/%{destination.port} duration %{_temp_.duration_hms} forwarded bytes %{network.bytes}"
```

Comment (andrewkroh) [reply x-pack/filebeat/module/cisco/shared/ingest/asa-ftd-pipeline.yml]: This does not match any of the samples we have
```
module/cisco//asa/test/additional_messages.log:May  5 19:02:58 dev01: %ASA-6-302023: Teardown stub TCP connection for fw111:10.10.10.10/39210 to net:192.168.2.2/10051 duration 0:00:00 forwarded bytes 0 Cluster flow with CLU closed on owner
module/cisco//asa/test/additional_messages.log:May  5 19:02:58 dev01: %ASA-6-302023: Teardown stub TCP connection for net:10.10.10.10/10051 to unknown:192.168.2.2/39222 duration 0:00:00 forwarded bytes 0 Forwarding or redirect flow removed to create director or backup flow
```

There's _reason_ string at the end. https://www.cisco.com/c/en/us/td/docs/security/firepower/Syslogs/b_fptd_syslog_guide/syslogs3.html#con_8182943

```suggestion
      pattern: "Teardown stub %{network.transport} connection for %{_temp_.cisco.source_interface}:%{source.address}/%{source.port} to %{_temp_.cisco.destination_interface}:%{destination.address}/%{destination.port} duration %{_temp_.duration_hms} forwarded bytes %{network.bytes} %{event.reason}"
```

**Comment by andrewkroh** [comment]
I pushed an update to the dissect pattern.

run tests

**Comment by andrewkroh** [comment]
@chifu1234 Can you please sign the CLA. https://www.elastic.co/contributor-agreement

**Comment by chifu1234** [comment]
@andrewkroh thxs will sign today

**Comment by andrewkroh** [comment]
@chifu1234 Hi, checking in to see if you could please sign the CLA. Then I'll get this merged. Thanks.

**Comment by chifu1234** [comment]
@andrewkroh sorry i did sign the CLA now.

**Comment by andrewkroh** [comment]
jenkins, run tests

**Review by andrewkroh — APPROVED** [review]

**Pull request merged**